bouncer/util.py

```diff
@@ -44,15 +53,37 @@ def parse_document(document):
     annotation_id = document["_id"]
     annotation = document["_source"]
 
+    # And that annotations always have "group" and "shared"
+    group = annotation["group"]
+    is_shared = annotation["shared"] is True
+
+    can_reveal_metadata = is_shared and group == "__world__"
+
     document_uri = None
+    quote = None
+    text = annotation.get('text', make_boilerplate_text())
+    if text == "":
+        text = make_boilerplate_text()
```

Comment:
This seems to be a bit of a confusing way of setting the default multiple times. You can just do:

```
text = annotation.get('text')
if not text:
    text = ANNOTATION_BOILERPLATE_TEXT
```

This works because:
* `{'foo': 'bar'}.get('key')` returns `None` which is falsy.
* An empty string (`''`) is falsy as well.

(See comment further below for the use of a constant instead of a function)